src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.data.person;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
```

Comment:
The convention is to name the test class based on the class being tested. e.g. `PersonTest`